index.js

```diff
@@ -362,10 +362,10 @@ function processVideoFormats(formats, isStream) {
 function videoTrackSort(a, b) {
   // Prefer English audio
   const englishAudioTag = 'original:lang%3Den';
-  if (a.url?.includes(englishAudioTag) && !b.url?.includes(englishAudioTag)) {
+  if ((a.url && a.url.includes(englishAudioTag)) && (!b.url || !b.url.includes(englishAudioTag))) {
```

Comment:
I was thinking more `!(b.url && b.url.includes(englishAudioTag))` but this works